index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone_project</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,900;1,100&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="nav-bar">
+                <a href="index.html"><img src="./images/Ccsummit2015_logo.png"></a>
+                <ul class="nav-menu">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Program</a></li>
+                    <li><a href="#">Join</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                    <li><a href="#">News</a></li>
+                </ul>
+                <button class="campaign"><a href="#">Campaign</a></button>
+            </div>
+            <div class="hamburguer">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
```

Comment:
- [ ] Good effort :+1: but there are some issues like the spacing between links is too much and also the last link in the microverse provided a design with the last link is a red colour with a border but in your case when I hover it then it change to the red colour so kindly fix these issues :slightly_smiling_face: check below :point_down: comparison between your design and Behance design :+1: 

**Your Design**
![Screenshot from 2023-04-20 04-07-46](https://user-images.githubusercontent.com/45226848/233226161-8d34fbdc-78fd-4249-9e22-32e217ed6e06.png)

**Behance Deisgn**
![Screenshot from 2023-04-20 04-08-08](https://user-images.githubusercontent.com/45226848/233226171-59c1b7ba-b3e2-4dd2-bf5c-0582564fe517.png)